### PR TITLE
PMM-9087: independent telemetry

### PR DESCRIPTION
**Pull request by fabio-silva**
**What this PR does / why we need it**: Telemetry/IA/STT toggles are independent from each other.

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-9087

FB: https://github.com/Percona-Lab/pmm-submodules/pull/2132

### CODE REVIEW

**Review by solovevayaroslavna — APPROVED** [review]

File: public/app/percona/settings/Settings.messages.ts [hunk 1]
```diff
@@ -13,6 +13,8 @@ export const Messages = {
     telemetryLabel: 'Telemetry',
     telemetryLink: `${GUI_DOC_URL}#server-admin-gui-telemetry`,
     telemetryTooltip: 'Option to send usage data back to Percona to let us make our product better.',
+    telemetryDisclaimer:
+      'Disabling Telemetry while Integrated Alerting/STT is on will inactivate intelligence (Downloading check and Alert templates).',
```
Comment (tiagomotasantos) [inline public/app/percona/settings/Settings.messages.ts:17]: Is "Integrated Alerting" correct here or should it be just "Alerting"?

File: public/app/percona/settings/components/Advanced/Advanced.tsx [hunk 2]
```diff
@@ -179,11 +180,13 @@ export const Advanced: FC<AdvancedProps> = ({
               tooltip={telemetryTooltip}
               tooltipLinkText={tooltipLinkText}
               link={telemetryLink}
-              className={cx({ [styles.switchDisabled]: values.stt || values.alerting })}
-              disabled={values.stt || values.alerting}
               dataTestId="advanced-telemetry"
               component={SwitchRow}
             />
+            <div className={styles.infoBox}>
+              <Icon name="info-circle" size={'xl'} className={styles.infoBoxIcon} />
```
Comment (tiagomotasantos) [inline public/app/percona/settings/components/Advanced/Advanced.tsx:187]:
```suggestion
              <Icon name="info-circle" size="xl" className={styles.infoBoxIcon} />
```

File: public/app/percona/settings/components/Advanced/Advanced.tsx [hunk 3]
```diff
@@ -255,8 +256,8 @@ export const Advanced: FC<AdvancedProps> = ({
             </div>
             <fieldset className={styles.technicalPreview}>
               <legend>{technicalPreviewLegend}</legend>
-              <div className={styles.technicalPreviewDoc}>
-                <Icon name="info-circle" size={'xl'} className={styles.technicalPreviewIcon} />
+              <div className={styles.infoBox}>
+                <Icon name="info-circle" size={'xl'} className={styles.infoBoxIcon} />
```
Comment (tiagomotasantos) [inline public/app/percona/settings/components/Advanced/Advanced.tsx:260]:
```suggestion
                <Icon name="info-circle" size="xl" className={styles.infoBoxIcon} />
```

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: public/app/percona/settings/components/Advanced/Advanced.tsx (hunk 3 above)
Comment (fabio-silva) [reply public/app/percona/settings/components/Advanced/Advanced.tsx]: 👍

File: public/app/percona/settings/components/Advanced/Advanced.tsx (hunk 2 above)
Comment (fabio-silva) [reply public/app/percona/settings/components/Advanced/Advanced.tsx]: 👍

File: public/app/percona/settings/Settings.messages.ts (hunk 1 above)
Comment (fabio-silva) [reply public/app/percona/settings/Settings.messages.ts]: Yep, shouldn't be :D

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**